geometry_msgs/Point[] surface1
geometry_msgs/Point[] surface2
geometry_msgs/Point[] surface3
geometry_msgs/Point[] surface4
geometry_msgs/Point[] surface5
mavros_msgs/MountControl[] gimbal
float64[] length
